src/import_edf.jl

```diff
@@ -639,7 +646,7 @@ function onda_samples_from_edf_signals(target::SamplesInfoV2, edf_signals,
                                           edf_signal.samples)
             encoded_samples = Onda.encode(sample_type(target), target.sample_resolution_in_unit,
                                           target.sample_offset_in_unit, decoded_samples,
-                                          missing)
+                                          dither_storage)
```

Comment:
```suggestion
                            dither_storage)
             catch e
                 if e isa DomainError
                     @warn "DomainError during `Onda.encode` can be due to a dithering bug; try calling with `dither_storage=nothing` to disable dithering."
                 end
                 rethrow()
             end
```

let's add an informative warning too so that people don't keep having to go down this wild goose chase...

Reply:
(had to do this as two suggestions because of the deleted line :(